opensearch_py_ml/sentence_transformer_model/sentencetransformermodel.py

```diff
@@ -743,6 +746,95 @@ def set_up_accelerate_config(
         with open(file_path, "w") as file:
             yaml.dump(default_file, file)
 
+    def make_model_config_json(
+        self,
+        model_name: str = None,
+        version_number: int = 1,
+        embedding_dimension: int = 384,
+        all_config: dict = None,
+        model_type: str = None,
+    ) -> None:
+        """
+        parse from config.json file of pre-trained hugging-face model to generate a model_config.json file. If all required
+        fields are given by users, use the given parameters and will skip reading the config.json,
+
+        Parameters
+        ----------
+        model_name: str = None
+            Optional, The name of the model. If None, default to parse from model id, for example,
+            'msmarco-distilbert-base-tas-b'
+        version_number: int = 1,
+            Optional, The version number of the model. If None, default to be 1.
+        embedding_dimension: int = 384,
```

Comment:
I made 384 because that was from opensearch website example [msmarco-distilbert-base-tas-b](https://opensearch.org/docs/latest/ml-commons-plugin/api/#example), but to match with the default, aligned and changed to 768

Reply:
In that example, `all-MiniLM-L6-v2` was used whose `embedding_dimension` is 384

Reply:
changed to 768